src/libsync/theme.cpp

```diff
@@ -132,6 +132,29 @@ QUrl Theme::stateOfflineImageSource() const
     return imagePathToUrl(themeImagePath("state-offline", 16));
 }
 
+QString Theme::trayHeaderThemeImagePath(const QString &iconName) const {
+    const bool headerHasDarkBg = isDarkColor(wizardHeaderBackgroundColor());
```

Comment:
@camilasan 
you can make the subsequent "arg()" parsing more readable if you just create `const QString iconColor = isDarkColor(wizardHeaderBackgroundColor() ? : "white" : "black";` and then just use "iconColor" in ".arg()".